gencode/main.ml

```diff
@@ -2,6 +2,28 @@
 
 open Lwt.Infix
 
+type emoji = {
+  code_point  : string;
+  emoji       : string;
+  bytes       : string;
+  name        : string;
+  description : string;
+}
+
+let rec emojis_list codes emojis bytes lets descs =
```

Comment:
Does my new commit improve it or did you mean something else? In the latter case please elaborate :)